packages/react-heartwood-components/src/components/Text/Text.js

```diff
@@ -1,16 +1,69 @@
 // @flow
-import React from 'react'
+import React, { Fragment } from 'react'
 import cx from 'classnames'
 
+import Button from '../Button/Button'
+import TextStyle from '../TextStyle/TextStyle'
+
 import type { Node } from 'react'
 
+// Components available for templating
+
+const TextComponentKey = {
+	style: TextStyle,
+	button: Button
+}
+
+// Allows basic templating functionality on text strings
+
+const TemplateEngine = (text = '', context = {}) => {
+	let re = /{{([^}}]+)?}}/g,
+		children = [],
+		cursor = 0,
+		match
+
+	let add = function(line, js) {
```

Comment:
js -> ?

`renderMethod` I think? Not sure what JS means here... other than JavaScript 😂